keras2onnx/_builtin.py

```diff
@@ -5,10 +5,19 @@
 ###############################################################################
 from .common.onnx_ops import apply_identity
 from .funcbook import set_converter
+from .ke2onnx.main import apply_reshape
```

Comment:
why not import it as same as the first import line?

Reply:
done